recipes/openmathreasoning/configs/solution_sdg/tir-openmath.yaml

```diff
@@ -67,7 +78,7 @@ stages:
       server_gpus: 8
       server_nodes: 1
       num_random_seeds: ${num_random_seeds_to_generate}
-      num_chunks: 10  # since data is big, we are parallelizing it 10x (for each seed, so in total 80 jobs are scheduled)
+      # num_chunks: 10  # since data is big, we are parallelizing it 10x (for each seed, so in total 80 jobs are scheduled)
```

Comment:
why this change?

Reply:
Same as above: 
Was creating too many jobs even when this toy dataset is quite small. I can change the comment to say "can use num_chunks to parallelize"